package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "14.0.0",
+  "version": "14.1.0",
```

Comment:
This is not necessary, the auto-release process will bump this up.

Also, since we have a new major version for the API, do we not need to make this a Major release as well?

Reply:
done